src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/DocumentHelper.java

```diff
@@ -961,4 +961,25 @@ public static Address getRespondentAddressET3(RespondentSumType respondentSumTyp
 
     }
 
+    /**
+     * Utility method to null check a string value.
+     * @param value to be checked if null or contains data
+     * @return null if the value is null or empty or will return the value itself
+     */
+    public static String nullChecker(String value) {
```

Comment:
1. Can't we reuse something instead? Is there no library that we can use to do this? What about StringUtils.defaultIfEmpty
2. I don't think DocumentHelper is an appropriate place for this method or listNullChecker. They are generic utility methods with nothing that ties them specifically to documents. They belong in a different class.

Reply:
Replaced with defaultIsEmpty, generic utility methods removed from code